setup.sh

```diff
@@ -35,7 +27,13 @@ clean(){
 install(){
   echo 'Installing project dependencies...'
   npm install
-  bower install --config.interactive=false
+  npm install --save --save-exact box-sizing-polyfill@0.1.0 jquery.easing@1.3.2 normalize-css@2.3.1 jquery@1.11.3 normalize-legacy-addon@0.1.0
```

Comment:
Ah, yeah! The lines were added to my `package.json` *by that command*, after @contolini and I paired and discovered that the post-install hook for Capital Framework to bring those packages in is incompatible with shrinkwrapping. 😞 

Makes sense to remove the line now, though. Like you said, it's in `package.json` now!